editor/src/clj/editor/tile_map.clj

```diff
@@ -957,17 +986,19 @@
       (gl/gl-draw-arrays gl GL2/GL_QUADS 0 (count vbuf)))))
 
 (defn- render-palette-active
-  [^GL2 gl render-args tile-source-attributes palette-tile]
-  (when palette-tile
-    (let [n palette-tile
-          w (:width tile-source-attributes)
-          h (:height tile-source-attributes)
-          x (mod n (:tiles-per-row tile-source-attributes))
-          y (int (/ n (:tiles-per-row tile-source-attributes)))
-          x0 (* x (+ tile-border-size w))
-          x1 (+ x0 w tile-border-size)
-          y0 (* y (+ tile-border-size h))
-          y1 (+ y0 h tile-border-size)
+  [^GL2 gl render-args tile-source-attributes start-tile end-tile]
```

Comment:
if we know two corners of the brush it's enough to draw the frame, similar what we do in `make-brush-from-selection-in-palette`